app/views/applications/_form.html.haml

```diff
@@ -0,0 +1,105 @@
+          = f.input :past_schedule, label: "Wykaz realizacji wnioskowanego projektu w latach poprzednich"
+          = f.input :collaboration, label: "Współpraca organizacyjna z innymi podmiotami"
+          = f.input :target_group, label: "Docelowa grupa odbiorców projektu"
+          = f.input :sponsor_enlistment, label: "Opis strategii pozyskiwania sponsorów"
+          = f.input :promotion_plans, label: "Planowane działania promocyjne projektu"
+        .row
+          %h4.text-center Doświadczenie zespołu organizatorów w podobnych projektach:
+          = f.simple_fields_for :experiences do |ex|
+            / TODO can this div be a row? layout dillemas
+            .experiences_row
+              .row
+                .col-md-4
+                  = ex.input :project_name, label: "Nazwa Projektu"
+                .col-md-4
+                  = ex.input :role, label: "Pełniona funkcja"
+                .col-md-4
+                  = ex.input :budget, input_html: { min: '0',
+                    required: true,  oninput: "this.value = Math.abs(this.value)",
+                    placeholder: 0},
+                    label: "Budżet"
+              .row
+                .col-md-6
+                  = ex.input :project_date, label: "Data wydarzenia"
+                .col-md-6
+                  = ex.input :member_name, label: "Imię i nazwisko członka zespołu"
+              %button.btn.btn-default.remove_row{type: "button"} Usuń
+          %button.btn.btn-default#add_experience{ type: "button" } Dodaj
+        .row
+          %h4.text-center Harmonogram realizacji projektu
+          = f.simple_fields_for :schedule_items do |si|
+            .row.schedule_items_row
+              .col-md-4
+                = si.input :todo, label: "Pozycja", input_html: { class: "disable_empty" }
+              .col-md-4
+                = si.input :start_date, label: "Data rozpoczęcia"
+              .col-md-4
+                = si.input :due_date, label: "Data zakończenia"
+              %button.btn.btn-default.remove_row{type: "button"} Usuń
+          %button.btn.btn-default#add_schedule_item{ type: "button" } Dodaj
+          = f.input :current_state, label: "Aktualny stan zaawansowania projektu lub powiązania z innymi projektami"
+        .row
+          %h4.text-center Skład zespołu realizatorów projektu
+          = f.simple_fields_for :roles do |role|
+            .row.role_row
+              .col-md-4
+                = role.input :member_role, label: "Funkcja"
+              .col-md-4
+                = role.input :first_name, label: "Imię"
+              .col-md-4
+                = role.input :last_name, label: "Nazwisko"
+              .col-md-4
+              %button.btn.btn-default.remove_row{type: "button"} Usuń
+          %button.btn.btn-default#add_member{ type: "button" } Dodaj członka
```

Comment:
All this stuff that shows text should go to `config/locales/pl.yml` and you should use i18n helpers in Rails